src/engine/websites/DoujinDesu.ts

```diff
@@ -19,4 +18,20 @@ export default class extends DecoratableMangaScraper {
     public override get Icon() {
         return icon;
     }
-}
\ No newline at end of file
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+        let uri = new URL(chapter.Identifier, this.URI);
+        let request = new FetchRequest(uri.href);
+        let data = await FetchCSS(request, "main#reader");
+        const chapterid = data[0].dataset['id'];
+
+        uri = new URL('/themes/ajax/ch.php', this.URI);
+        request = new FetchRequest(uri.href, {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+            body: new URLSearchParams({ id: `${chapterid}` })
```

Comment:
I think string interpolation is redundant as URLSearchParams will convert everything into a string, so
`URLSearchParams({ id: chapterid })` should be sufficient